articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -181,7 +181,7 @@ table_service.delete_table('tasktable')
 Now that you've learned the basics of Table storage, follow these links
 to learn more.
 
-* [Python Developer Center](/develop/python/)
+* [Python Developer Center](https://azure.microsoft.com/en-us/develop/python/)
```

Comment:
@asears Hi Andrew, I'll be requesting that this PR be closed in one week if no further action is taken.